explainer/README.md

```diff
@@ -135,18 +130,18 @@ This discussion assumes that third-party cookies, like all other third party sto
 
 All fenced frame related functions will live in its own class, in the same way that iframe-related funcionality lives in HTMLIFrameElement.
 
-#### Opaque-Ads Can Load API
+#### Can Load API
 
-There are various reasons a fenced frame embedded as an opaque ad could refuse to load in a page. For example, if the page is not in a secure context, or if CSPEE is specified in the embedding frame, the fenced frame will refuse to load. This is a lot for a developer to keep track of.
+There are various reasons a fenced frame config could refuse to load in a page. For example, if the page is not in a secure context, or if CSPEE is specified in the embedding frame, the fenced frame config will refuse to load. This is a lot for a developer to keep track of.
 
 If the process of getting an ad in the page is complex or expensive, there needs to be a way to ensure that the resulting ad will actually end up in the page before the expensive process begins.
 
-A static API method will be introduced to the HTMLFencedFrameElement class to check this. No fenced frame will be created when calling this API, and it can be invoked before actually attempting to create a fenced frame. The API will return a boolean, true if an opaque-ads fenced frame would be able to load in the caller's context, false if not.
+A static API method will be introduced to the HTMLFencedFrameElement class to check this. No fenced frame will be created when calling this API, and it can be invoked before actually attempting to load a fenced frame config. The API will return a boolean, true if the config would be able to load in the caller's context, false if not.
 
 ##### Example usage
 
 ```
-HTMLFencedFrameElement.canLoadOpaqueURL();
+HTMLFencedFrameElement.canLoadConfig(config);
```

Comment:
Right now the API is still `canLoadOpaqueURL()`. Should we make a reference to that, saying that `canLoadConfig()` will replace it in the future, and in the meantime both will exist?

Reply:
We discussed maybe removing this section from the explainer for now. Do you still plan on doing that?

Reply:
I semi-reverted the changes here. Now it describes the same HTMLFencedFrameElement.canLoadOpaqueURL() function, but as "would a fenced frame config with an opaque mapped url be able to load in this context"?

We can fully replace it when the replacement is fully determined/implemented.